Move APIs for service pages (#3512)
viamrobotics/docs: docs/appendix/apis/services/SLAM.md

---
title: "SLAM service API"
linkTitle: "SLAM"
weight: 60
type: "docs"
tags: ["slam", "services"]

> description: "The SLAM service API allows you to get machine's position within a map."

Author: ```suggestion
description: "Give commands to get a machine's position within a map."
```

icon: true
images: ["/services/icons/slam.svg"]
---

> The SLAM service API allows you to get machine's position within a map.

Author: ```suggestion
The SLAM service API allows you to get a machine's position within a map.
```
